WalletWasabi.Fluent/ViewModels/Login/LoginViewModel.cs

```diff
@@ -52,7 +54,18 @@ public LoginViewModel(WalletViewModelBase walletViewModelBase)
 					wallet.Login();
 
 					// TODO: navigate to the wallet welcome page
-					Navigate().To(walletViewModelBase, NavigationMode.Clear);
+					var navigateTo = walletViewModelBase;
```

Comment:
Nit: Could be renamed to `targetViewModel`

Reply:
Ack.